### PR TITLE
feat: add xml to prod for landsat v2

### PR DESCRIPTION
**Pull request by tdivoll**
This PR adds the tested XML to the production metadata so that I can pull the changes and update the repo on the production server!

### CODE REVIEW

File: content/erddap/datasets.xml [hunk 1]
```diff
@@ -5296,6 +5296,511 @@ tke:                 Rad    Rad    Rad    Clo</att>
...
+        <att name="institution">Rhode Island Data Discovery/United States Geological Survey</att>
+        <att name="publication">https://doi.org/10.26300/ja0b-xa86</att>
+        <att name="references">Rhode Island Data Discovery/United States Geological Survey</att>
+        <att name="source">https://earthexplorer.usgs.gov/ accessed through https://code.earthengine.google.com/</att>
+        <att name="title">Landsat Satellite Surface Temerapture version 2, Mt. Hope Bay</att>
+    </sourceAttributes -->
```
Comment (eldu) [inline content/erddap/datasets.xml:5323]: This is commented out, do we want it back?

File: content/erddap/datasets.xml [hunk 2]
```diff
@@ -5296,6 +5296,511 @@ tke:                 Rad    Rad    Rad    Clo</att>
...
+    <axisVariable>
+        <sourceName>time</sourceName>
+        <destinationName>time</destinationName>
+        <!-- sourceAttributes>
+            <att name="_ChunkSizes" type="int">764</att>
+            <att name="long_name">Time</att>
+            <att name="max" type="double">1.6644096E9</att>
+            <att name="min" type="double">4.52304E8</att>
+            <att name="units">seconds since 1970-01-01T00:00:00Z</att>
+        </sourceAttributes -->
```
Comment (eldu) [inline content/erddap/datasets.xml:5350]: Same question here?

**Review by eldu — COMMENTED** [review]
Hmm, there is quite a bit of commented out code. Perhaps remove it or add in a comment as to why you're leaving it commented out?

**Comment by tdivoll** [comment]
@eldu I cleaned up all the commented out code

File: content/erddap/datasets.xml [hunk 3]
```diff
@@ -5296,6 +5296,424 @@ tke:                 Rad    Rad    Rad    Clo</att>
...
+    </addAttributes>
+    <axisVariable>
+        <sourceName>time</sourceName>
+        <destinationName>time</destinationName>
+        <!-- sourceAttributes>
+            <att name="_ChunkSizes" type="int">764</att>
+        </sourceAttributes -->
```
Comment (eldu) [inline content/erddap/datasets.xml:5339]: There are a few more about these Chunksizes in different places as well... Do we need this?

File: content/erddap/datasets.xml [hunk 4]
```diff
@@ -5296,6 +5296,424 @@ tke:                 Rad    Rad    Rad    Clo</att>
...
+        </addAttributes>
+    </axisVariable>
+    <axisVariable>
+        <sourceName>x</sourceName>
+        <destinationName>X</destinationName>
+        <addAttributes>
+            <att name="long_name">X</att>
+            <att name="source_name">x</att>
+            <!-- <att name="standard_name">longitude</att>
+            <att name="units">degrees_east</att> -->
```
Comment (eldu) [inline content/erddap/datasets.xml:5476]: This comments seems like we can bring back? or take out?

File: content/erddap/datasets.xml [hunk 5]
```diff
@@ -5296,6 +5296,424 @@ tke:                 Rad    Rad    Rad    Clo</att>
...
```
Comment (eldu) [inline content/erddap/datasets.xml:5683]: Same here

**Review by eldu — APPROVED** [review]
I added in some more comments. There is still some commented out stuff especially regarding chunk sizes. Also some are units which could maybe be brought in.

Approving in the meantime !

File: content/erddap/datasets.xml (hunk 3 above)
Comment (tdivoll) [reply content/erddap/datasets.xml:5339]: Looking at Mary's code, it seems like she left the chunk sizes as comments, maybe to help if we ever had to build the files again?

File: content/erddap/datasets.xml (hunk 4 above)
Comment (tdivoll) [reply content/erddap/datasets.xml:5476]: These are more of a hint that X and Y local coordinates correspond to longitude and latitude, but ERDDAP just needs the grid size to display the graphs properly.

**Pull request merged**